go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/benlaurie/objecthash v0.0.0-20180202135721-d1e3d6079fc1
 	github.com/fatih/color v1.13.0
 	github.com/flyteorg/flyteidl v1.3.14
-	github.com/flyteorg/flyteplugins v1.0.45
+	github.com/flyteorg/flyteplugins v1.0.48
```

Comment:
this should be 1.0.47 - https://github.com/flyteorg/flyteplugins/releases. 1.0.48 will pull in code deprecating the barrier transitions, which is not meant to be a part of this merge.

Reply:
done. thanks for the catch